epoch_test.go

```diff
@@ -286,6 +286,15 @@ func TestEpochBlockSentTwice(t *testing.T) {
 func TestEpochBlockTooHighRound(t *testing.T) {
 	l := makeLogger(t, 1)
 
+	var rejectedBlock bool
+
+	l.intercept(func(entry zapcore.Entry) error {
```

Comment:
Needs to be right after the creation, otherwise we override the reference when intercepting, while goroutines try to read the reference